telegram/_inline/inlinekeyboardbutton.py

```diff
@@ -62,6 +62,10 @@ class InlineKeyboardButton(TelegramObject):
 
         * After Bot API 6.1, only ``HTTPS`` links will be allowed in :paramref:`login_url`.
 
+    .. versionchanged:: 20.0
+       :attr:`web_app` is considered as well when comparing objects of this type in terms of
```

Comment:
```suggestion
    .. versionchanged:: 20.0
        :attr:`web_app` is considered as well when comparing objects of this type in terms of
```
same in other places so it renders right

Reply:
what about that doesn't render right for you? 🤔 this is how it looks like for me:

![image](https://user-images.githubusercontent.com/22366557/168469923-dd068063-4b8d-4cc2-bcaa-855d5f8411f6.png)

and the additional whitespace doesn't change anything about it …

Reply:
nvm, there was one sphinx version where that didn't render right